src/devices/cpu/m6502/gew7.h

```diff
@@ -0,0 +1,87 @@
+// license:BSD-3-Clause
+// copyright-holders:Devin Acker
+/***************************************************************************
+
+    gew7.h
+
+    Yamaha GEW7, GEW7I, GEW7S (65c02-based)
+
+***************************************************************************/
+#ifndef MAME_CPU_M6502_GEW7_H
+#define MAME_CPU_M6502_GEW7_H
+
+#include "m65c02.h"
+#include "sound/gew7.h"
+
+class gew7_device : public m65c02_device, public device_mixer_interface
+{
+public:
+
+	template<offs_t i> auto port_in_cb() { return m_in_cb[i].bind(); }
+	template<offs_t i> auto port_out_cb() { return m_out_cb[i].bind(); }
+
+	u8 timer_stat_r();
+	void timer_stat_w(u8 data);
+	void timer_en_w(u8 data);
+
+	u8 timer_count_r(offs_t offset);
+	void timer_count_w(offs_t offset, u8 data);
+
+	void bank_w(offs_t offset, u8 data);
+
+	u8 port_r(offs_t offset);
+	void port_w(offs_t offset, u8 data);
+	u8 port_ddr_r(offs_t offset);
+	void port_ddr_w(offs_t offset, u8 data);
+
+protected:
+	gew7_device(const machine_config& mconfig, device_type type, const char* tag, device_t* owner, uint32_t clock);
+
+	virtual void device_add_mconfig(machine_config& config) override;
```

Comment:
Please keep right-associative to the right of the space, like all the other things in this folder – `machine_config &mconfig`, etc.  It’s easier to misread it if you put it on the left of the space.

Consider:
```c++
int* a, b;
```
It misleadingly implies that `*` is left associative, applying to the type `int`, as if `a` and `b` were both type `int *`.

In contrast:
```c++
int *a, b;
```
It’s clear that the `*` is right-associative, applying to `a` only, and `b` has type `int`.